spec/spec_helper.rb

```diff
@@ -6,6 +6,8 @@
 require "bundler/setup"
 require "rspec"
 
+ENV["RAILS_ENV"] ||= "test"
+
```

Comment:
pls add `raise "Specs must be run in test environment" if ENV["RAILS_ENV"] != "test"`